1-Initial-Setup/1.8-GNOME-Display-Manager.bats

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bats
+
+@test "1.8.1 Ensure GNOME Display Manager is removed (Manual)" {
+    run bash -c "dpkg -s gdm3 | grep -E '(Status:|not installed)'"
+    [ "$status" -eq 1 ]
+}
+
+@test "1.8.2 Ensure GDM login banner is configured (Automated)" {
+    if dpkg -s gdm3; then
+        run bash -c "cat /etc/gdm3/greeter.dconf-defaults"
+        [ "$status" -eq 0 ]
+        [[ "$output" = *"[org/gnome/login-screen]"* ]]
+        [[ "$output" = *"banner-message-enable=true"* ]]
+        [[ "$output" = *"banner-message-text="* ]]
+    fi
+}
+
+@test "1.8.3 Ensure disable-user-list is enabled (Automated)" {
+    if dpkg -s gdm3; then
+        run bash -c "grep -E '^\s*disable-user-list\s*=\s*true\b' /etc/gdm3/greeter.dconf-defaults"
+        [ "$status" -eq 0 ]
+    fi
+}
+
+@test "1.8.4 Ensure XDCMP is not enabled (Automated)" {
+    run bash -c "grep -Eis '^\s*Enable\s*=\s*true' /etc/gdm3/custom.conf"
+    [ "$status" -ne 0 ]
```

Comment:
`-ne 0` could lead to a false negative. Is `-eq 2` more suitable?

Reply:
Why `2`? I guess ` -eq 1` should be right, don't you think?

Reply:
I testet it on my machine and got rc 2. I thought that would be the correct rc

Reply:
If there is no /etc/gdm3/custom.conf, the exit code is 2. If the string is not found in /etc/gdm3/custom.conf the exit code is 1. So I think we can keep `[ "$status" -ne 0 ]`